testsuite/tests/lib-dynlink-csharp/Makefile

```diff
@@ -30,7 +30,9 @@ all: prepare bytecode bytecode-dll native native-dll
 prepare:
 	@if $(SUPPORTS_SHARED_LIBRARIES); then \
```

Comment:
At least for now, to be consistent with the rest of this `Makefile`, could you add `|| ! which $(CSC) >/dev/null 2>&1;` (cf. ll. 39, 53, 70, 84) and remove the change below?

These tests are (at present) only targeting the Microsoft C Sharp compiler, so compiling anything is pointless on other systems.